config/helpers.ts

```diff
@@ -201,6 +203,7 @@ export const getFileMetadata = (file: string, root: string) => {
 
     return Observable.of<SnippetFileInput>({
         relativePath: relativePath,
+        type: /private-samples/.test(file) ? 'private' : 'public',
```

Comment:
This is unsafe.  What if I called a file "make-private-samples-worksheet.yaml".  If you're using a regular expression, be sure to specify its start and end conditions.  Or just use something else, e.g., an API to get the parent of the file, or to at least only get the path portion of the file rather than the file name.